test/setup/deployers/operator.ts

```diff
@@ -1,19 +1,53 @@
+import * as sleep from 'sleep-promise';
 import { IDeployer, IImageOptions } from './types';
 import * as kubectl from '../../helpers/kubectl';
 
+const PULL_KUBERNETES_OPERATOR_EVENT = 'docker.io/snyk/kubernetes-operator';
```

Comment:
Is there a little bit more info we can add to this event, not just the image name? Or you think this gives us enough confidence that we have the right event?

Reply:
What do you think if I include a comment showing an example of the event we're looking for? Because it's exactly the relevant information that we need to look for in the event to make sure that it has loaded the correct operator image.

Reply:
That works too 😄 Just not sure why not put the whole event object in the string, rather than a portion of the event and the whole event in a comment